src/NanopassSharp/Descriptions/BaseDescription.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace NanopassSharp.Descriptions;
+
+/// <summary>
+/// A simple implementation of <see cref="ITransformationDescription"/>.
```

Comment:
I feel like the docs are a bit lacking here. There's no reason you couldn't just say something like "A pair of pattern at transformation that together make up an ITransformationDescription."
The current doc saying "a simple implementation" tells nothing to the user.